src/server/src/main/java/io/cassandrareaper/management/http/models/Job.java

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright 2023-2023 DataStax, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.cassandrareaper.management.http.models;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.google.common.annotations.VisibleForTesting;
+import org.apache.cassandra.utils.progress.ProgressEventType;
+
+public class Job {
```

Comment:
NB: if this is because we want to have a model for the data returned in json form from the mgmt API side, please note that all models should be imported from the mgmt API client, not declared in Reaper.

Reply:
It's redeclared here (and probably commented also) since there's no mgmt-api client in the Reaper, thus there's nothing to use. Once that's in, then this class disappears.

Reply:
@burmanm the management API client is now available via this PR https://github.com/k8ssandra/management-api-for-apache-cassandra/pull/341

We should be able to eliminate the Job class and begin integration between mgmt API and  now I think. Let me know if you agree?